core/templates/dev/head/combined-tests.spec.ts

```diff
@@ -46,12 +46,14 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
-// Then we find all the tests. Note that known failing files are exempted;
+// Then we find all the tests and controller, directive, service/factory files
+// Files that shouldn't be tested are exempted (from services_sources folder)
```

Comment:
All files from the services_sources folder are exempted, because they shouldn't be tested (those files are just intended as data files for backend tests).

Reply:
Done